src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4590,6 +4590,7 @@ internal static SessionStateAliasEntry[] BuiltInAliases
                     // These were transferred from the "transferred from the profile" section
                     new SessionStateAliasEntry("cat", "Get-Content"),
                     new SessionStateAliasEntry("cp", "Copy-Item", string.Empty, AllScope),
+                    new SessionStateAliasEntry("lp", "Out-Printer"),
```

Comment:
The alias conflicts with Unix command. Do we really want the alias?

Reply:
Let's have @PowerShell/powershell-committee make a call here

Reply:
@PowerShell/powershell-committee reviewed this and agrees to remove the alias